app.py

```diff
@@ -1,11 +1,15 @@
-import flask
+"""
+The entry point of the BotTrition app.
+"""
+
 import os
 import json
+import flask
 
 from flask_login import UserMixin
 from flask_login import login_user, current_user, LoginManager
 from flask_login.utils import login_required
-from flask_sqlalchemy import SQLAlchemy
+from database import db
```

Comment:
Good job importing from another file to prevent possible circular imports in the future!